blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -88,23 +88,24 @@ export class Activity extends Component {
         }
 
         const latestRun = runs[0];
+        const head = 'pipelinedetail.activity.header';
```

Comment:
why not move this to top level and use

```
t(`${head}.xxx`)
```

all the time (especially the one above)